### DISCUSSION TITLE
DatePicker

### DISCUSSION THREAD

**Discussion by eduardocque** [chakra-ui/ark#2969] [General]
### Description

after major release now fails
![image](https://github.com/user-attachments/assets/caf1af48-f9df-4a22-a1ee-29f84cb92fbc)

### Link to Reproduction (or Detailed Explanation)

dont have

### Steps to Reproduce

after upgrade from V3 to V4, even that i dont have setup (value / min / max / defaultValue) crash a lot with invalid date or this one in the screenshot

### Ark UI Version

4.1.0

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Chrome

**Comment by eduardocque** [comment]
based on the documentation min/max should be DateValue or undefined
<img width="473" alt="image" src="https://github.com/user-attachments/assets/24ad01d6-eb18-409b-b13f-605a52864aa2">

but somewhere in the code is setting up the variables as empty string
![image](https://github.com/user-attachments/assets/31a1dd49-9cfa-4568-bdb0-fcaec13c951b)


to be sure im not making something wrong im not even passing those values
<img width="375" alt="image" src="https://github.com/user-attachments/assets/3306f63c-000a-478d-a2d2-66ccc35f3589">

**Reply by segunadebayo** [reply]
Can you attach a reproduction?

**Comment by segunadebayo** [comment]
Closing — DatePicker is fully functional with ranges, presets, and multiple views. Check the docs for examples.
